packages/build-tools/cmd/main.go

```diff
@@ -18,5 +18,5 @@ func init() {
 }
 
 func Execute() {
-	cobra.CheckErr(rootCmd.Execute())
+	rootCmd.Execute()
```

Comment:
Why did you remove the error check? This essentially silences errors on *all* tool commands in *all* cases. This *does* seemingly fix `tool rm -f ...` but that hopefully isn't what you were trying to do with this change. It also doesn't actually fix `tool rm -f` since an error will still cause it to abort removing files which means parameters after a missing file won't be deleted.

Reply:
So unless I'm very mistaken, there's already an implicit `CheckErr` in `Execute`. Before this change, an error would result in the following:

```
Error: error message
Usage: help message for the tool
...
...
Error: error message
```

After this change, the duplicate second error message is dropped, so it's just `Error:`, `Usage:`. Am I missing a case where this is necessary to print the error message?

Reply:
Without `CheckErr()`, the process always exits with code 0. If you just want to remove the duplicate error message, you should change that call to the following:
```go
err := rootCmd.Execute()
if err != nil {
  os.Exit(1)
}
```